code/index.html

```diff
@@ -1,22 +1,180 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lexend+Giga&family=Noto+Sans+JP:wght@300&family=Oswald&family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <header>
+    <img class="logo" src="./images/logo.svg" />
```

Comment:
Don't forget to add an alt tag for images, describing the image. This is what will be read for people using screen readers that might be visually impaired and can't see the image 👓👀